README.md

```diff
@@ -0,0 +1,143 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>README Template</b> <h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello-Microverse] <a name="about-project"></a>
+
+> A simple HTML project, created in block 1.
+
+**[Hello-Microverse]** is a test to see if I understand how to create a professional repository.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><p>HTML</p></li>
+    <li><p>CSS</p></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- An HTML file with a heading
+- A CSS file with a color style
+- A fancy README file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+    A web browser
+
+### Setup
+
+Clone this repository to your desired folder:
+    https://github.com/jlvFlores/Hello-Microverse.git
+
+### Install
+
+Install this project with:
+    gitBash
+
+### Usage
+
+To run the project, execute the following command:
+    Press the Go Live button 
+
+### Run tests
+
+To run tests, run the following command:
+    Press the Go Live button 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@jlvFlores](https://github.com/jlvFlores)
+- Twitter: [@JoseVaz44312762](https://twitter.com/JoseVaz44312762)
+- LinkedIn: [Jose Vazquez](https://www.linkedin.com/in/jose-vazquez-178a8225a/)
+
+👤 **Author2**
+
+- GitHub: [@theShadow](https://github.com/andilemashala)
+
+👤 **Author3**
+
+- GitHub: [@Javier Salinas](https://github.com/javier-salinas)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] KIndly note that you are the only author as this is a solo project. Please remove other authors you added 👍🏽